package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "@k-phoen/backstage-plugin-grafana",
+  "name": "@spreadshirt/backstage-plugin-grafana",
   "version": "0.1.12",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
   "private": false,
-  "homepage": "https://github.com/K-Phoen/backstage-plugin-grafana",
+  "homepage": "https://github.com/spreadshirt/backstage-plugin-grafana",
   "bugs": {
-    "url": "https://github.com/K-Phoen/backstage-plugin-grafana/issues"
+    "url": "https://github.com/spreadshirt/backstage-plugin-grafana/issues"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/K-Phoen/backstage-plugin-grafana.git"
+    "url": "https://github.com/spreadshirt/backstage-plugin-grafana.git"
```

Comment:
Please split this into two commits, one for updating the urls to point to this repo, and one to update the dependencies for Backstage 1.4.0.

Reply:
Done